nucypher/cli/commands/ursula.py

```diff
@@ -295,7 +296,8 @@ def ursula():
 @option_force
 @option_config_root
 @group_general_config
-def init(general_config, config_options, force, config_root):
+@click.option('--secret-key', help="An custom pre-secured secret hex blob to use for key derivations", type=click.STRING)
```

Comment:
I have reservations about calling it `secret_key`, seeing how we already have `SecretKey` objects, and that's not it. "Key seed" or "key material" perhaps?

Reply:
✅